src/Workspaces/Remote/ServiceHub/Host/RemoteWorkspace.SolutionCreator.cs

```diff
@@ -381,35 +367,35 @@ private async Task<Project> UpdateProjectInfoAsync(Project project, Checksum inf
                 Func<Solution, ImmutableArray<DocumentId>, Solution> removeDocuments,
                 CancellationToken cancellationToken) where TDocumentState : TextDocumentState
             {
-                using var olds = SharedPools.Default<HashSet<Checksum>>().GetPooledObject();
```

Comment:
note: i'm not changing this method's semantics at all (either in this pr or in the cone sync pr).  but i at least wanted the patterns and naming to be in sync with the project syncing part.